### DISCUSSION TITLE
GitHub REST API response missing field

### DISCUSSION THREAD

**Discussion by digineo1** [community/community#24710] [Apps, API and Webhooks]
<p>Response from the REST API when calling the <code>/user/starred</code> endpoint is missing the <code>topics</code> field which should be expected, according to the documentation located here: <a href="https://docs.github.com/en/rest/reference/activity#list-repositories-starred-by-the-authenticated-user" rel="noopener nofollow ugc">Documentation</a></p>

**Comment by gm3dmo** [comment, marked as answer]
<p>The example in the documentation looks like it uses a repo listing that has been generated where the <a href="https://docs.github.com/en/rest/overview/api-previews#repository-topics" rel="noopener nofollow ugc"><code>mercy</code></a> preview header has been passed in the request.</p>
<p>Please try adding the <code>mercy</code> preview header to your request. I’ve provide an example using <code>curl below</code>:</p>
<pre><code class="lang-auto">curl \
     -H "Accept: application/vnd.github.v3+json" \
     -H "Accept: application/vnd.github.mercy-preview+json" \
     -H "Authorization: token ${GITHUB_TOKEN}" \
        https://api.github.com/user/starred

</code></pre>
<p>for me it returns the <code>topics</code> field:</p>
<pre><code class="lang-auto">    "open_issues_count": 2,
    "license": null,
    "topics": [
      "cats",
      "dogs",
    ],
    "forks": 0,
    "open_issues":
</code></pre>

**Comment by digineo1** [comment]
<p>Thank you. That worked for me.</p>
